example/sofast_fringe/example_calibration_screen_shape.py

```diff
@@ -62,27 +73,16 @@ def run_screen_shape_calibration(save_dir):
     display_shape = cal.as_DisplayShape('Example display shape')
 
     # Save DisplayShape file
-    file = join(save_dir, 'display_shape.h5')
+    file = join(dir_save, 'display_shape.h5')
     display_shape.save_to_hdf(file)
     lt.info(f'Saved DisplayShape file to {file:s}')
 
     # Save calibration figures
     for fig in cal.figures:
-        file = join(save_dir, fig.get_label() + '.png')
+        file = join(dir_save, fig.get_label() + '.png')
         lt.info(f'Saving figure to: {file:s}')
         fig.savefig(file)
 
 
-def example_driver():
```

Comment:
Did you mean to delete this method? If yes, then great! I just wanted to make sure you're aware of it.

Reply:
Yes, since the `example_driver()` now only directly called `example_calibation_screen_shape()` I just called that directly.